scanpipe/pipes/purldb.py

```diff
@@ -320,3 +326,94 @@ def populate_purldb_with_discovered_dependencies(project, logger=logger.info):
         chunk_size=10,
         logger=logger,
     )
+
+
+def send_project_json_to_matchcode(
+    project, timeout=DEFAULT_TIMEOUT, api_url=PURLDB_API_URL
+):
+    """
+    Given a `project`, create a JSON scan of the `project` CodebaseResources and
+    send it to PurlDB for matching. Return a tuple containing strings of the url
+    to the particular match run and the url to the match results.
+    """
+    scan_output_location = to_json(project)
+    with open(scan_output_location, "rb") as f:
+        files = {"upload_file": f}
+    response = request_post(
+        url=f"{api_url}matching/",
+        data={},
+        timeout=timeout,
+        files=files,
+    )
+    run_url = response["runs"][0]["url"]
+    url = response.get("url")
+    results_url = url + "results/"
+    return run_url, results_url
```

Comment:
I think we should keep the `results_url` out of this function top return a single value and create the create the `results_url`  in `match_to_purldb`

Reply:
I'm not sure how to get the results URL without saving the URL to the project from the initial POST response, like I do here. I could just return the entire response.

Reply:
I see I can get the project url from the run response.